DOCS/man/input.rst

```diff
@@ -1429,8 +1429,8 @@ IPC sees. Note that the C API has separate C-level declarations with
 ``mpv_event``, which may be slightly different.
 
 Note that events are asynchronous: the player core continues running while
-events are delivered to scripts and other clients. In some cases, you can hooks
-to enforce synchronous execution.
```

Comment:
This should be in its own commit in another PR, not here.

Reply:
This change is still in the commit somehow. I think you should probably rebase the entire branch against master again and get rid of the conflict.

Reply:
Gah why does github hide unresolved conversations? Stupid. I'll make a new one.